components/Layout/layout.tsx

```diff
@@ -3,11 +3,13 @@ import scss from './layout.module.scss';
 
 type LayoutProps = { children: React.ReactNode };
 const Layout = ({children}: LayoutProps): JSX.Element => (
-    <div className={scss.layout}>
+    <main className={scss.layout}>
```

Comment:
the `header` shouldn't be inside the `main` though.

We should have 
```html
<body>
    <header/>
    <main/>
    <footer/>
</body>
```

so 
- the `body` is everything visible on the page (ie not the scripts and stylesheets or metadata etc)
- the `main` is the content that's the 'purpose' of the page - ie not the header and footer etc)